package.json

```diff
@@ -28,7 +28,12 @@
     "start:viewer": "npm run start:pkg -- @bpmn-io/form-js-viewer",
     "start:viewer:stress": "cross-env SINGLE_START=stress npm run dev -- @bpmn-io/form-js-viewer",
     "start:editor": "npm run start:pkg -- @bpmn-io/form-js-editor",
-    "start:playground": "npm run start:pkg -- @bpmn-io/form-js-playground"
+    "start:playground": "npm run start:pkg -- @bpmn-io/form-js-playground",
+    "dev:e2e": "vite",
+    "test:visual": "playwright test visual",
+    "build:e2e": "vite build",
```

Comment:
This gives me an error on my arm64 macOS machine

```text
> vite build

failed to load config from /work/vite.config.js
error during build:
Error:
You installed esbuild for another platform than the one you're currently using.
This won't work because esbuild is written with native code and needs to
install a platform-specific binary executable.

Specifically the "@esbuild/darwin-arm64" package is present but this platform
needs the "@esbuild/linux-arm64" package instead. People often get into this
situation by installing esbuild on Windows or macOS and copying "node_modules"
into a Docker image that runs Linux, or by copying "node_modules" between
Windows and WSL environments.

If you are installing with npm, you can try not copying the "node_modules"
directory when you copy the files over, and running "npm ci" or "npm install"
on the destination platform after the copy. Or you could consider using yarn
instead of npm which has built-in support for installing a package on multiple
platforms simultaneously.

If you are installing with yarn, you can try listing both this platform and the
other platform in your ".yarnrc.yml" file using the "supportedArchitectures"
feature: https://yarnpkg.com/configuration/yarnrc/#supportedArchitectures
Keep in mind that this means multiple copies of esbuild will be present.

Another alternative is to use the "esbuild-wasm" package instead, which works
the same way on all platforms. But it comes with a heavy performance cost and
can sometimes be 10x slower than the "esbuild" package, so you may also not
want to do that.

    at generateBinPath (/work/node_modules/esbuild/lib/main.js:1874:17)
    at esbuildCommandAndArgs (/work/node_modules/esbuild/lib/main.js:1955:33)
    at ensureServiceIsRunning (/work/node_modules/esbuild/lib/main.js:2119:25)
    at build (/work/node_modules/esbuild/lib/main.js:2011:26)
    at bundleConfigFile (file:///work/node_modules/vite/dist/node/chunks/dep-24daf00c.js:64347:26)
    at loadConfigFromFile (file:///work/node_modules/vite/dist/node/chunks/dep-24daf00c.js:64323:31)
    at resolveConfig (file:///work/node_modules/vite/dist/node/chunks/dep-24daf00c.js:63932:34)
    at build (file:///work/node_modules/vite/dist/node/chunks/dep-24daf00c.js:46246:26)
    at CAC.<anonymous> (file:///work/node_modules/vite/dist/node/cli.js:812:15)
````

Reply:
@pinussilvestrus Added a new note [here](https://github.com/bpmn-io/form-js/pull/632/files#diff-d9770bfbd9e3f53581cb560341b3ab0afe2fae65ab497e8d05f2877a429513e1R14)

I tested it on my Mac M2 and it worked. Let me know what you think